actnow/petitions/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import include, path
+from rest_framework import routers
+
+from .views import PetitionSignatureView, PetitionView
+
+router = routers.DefaultRouter()
+router.register(r"", PetitionView)
+router.register(r"signatures", PetitionSignatureView)
```

Comment:
One common case that I see is showing a petition with signatories (Kind of like `/petitions/<id>/signatures`). How are we handling this?